wrapper/src/main/java/software/amazon/jdbc/HikariPooledConnectionProvider.java

```diff
@@ -187,14 +188,16 @@ public Connection connect(
       @NonNull HostSpec hostSpec,
       @NonNull Properties props)
       throws SQLException {
+    final Properties copiedProps = PropertyUtils.copyProperties(props);
+    PropertyUtils.addConnAttrForClientIdentification(dialect, copiedProps);
```

Comment:
Need similar changes as for DriverConnectionProvider/DataSourceConnectionProvider